chirp/drivers/ga510.py

```diff
@@ -278,6 +426,30 @@ def get_features(self):
                           (400000000, 480000000)]
         return rf
 
+    def validate_memory(self, mem):
+        msgs = chirp_common.CloneModeRadio.validate_memory(self, mem)
```

Comment:
This is python2 syntax, which still works (for now) but you should not add new code with this old syntax. The new way is:
```
msgs = super().validate_memory(mem)
```
The code you're moving has it right, so this is regressing.

Also, please don't move this, as it means that (a) it looks like you're making a big change here in history and (b) it makes it harder to tell *if* you're changing anything. If this is different from the `validate_memory()` you're removing/moving below, please make the changes there so the diff shows what you're actually changing.